concept-anchor.md

```diff
@@ -0,0 +1,51 @@
+
+## Problem
+Often InnerSource is a term that's novel to organizations, making it often difficult to start a conversation with management about adopting InnerSource.
+
+## Context
+An individual sees an opportunity for InnerSource to make an impact within an organization. They would like to discuss InnerSource with management. Being a relatively traditional and bureaucratic company, it is difficult to broach the subject of InnerSource with management as they are reluctant to engage on a topic that they've neither heard of nor know anything about.
+
+These organizations might have an existing strategy, such as Agile, Lean, or DevOps transformation. To gain mindshare, it might be useful to start with these existing, well known concepts and build InnerSource on top of these concepts.
+
+## Forces
+* Management are reluctant to talk to you unless you can tie projects in to an existing corporate strategy.
+* Starting to explain InnerSource may be difficult unless one starts with a point decision makers are familiar with.
+* Agile/DevOps/Lean have already been heavily marketed and are commonly discussed terms in corporations, conferences, magazines etc.
+
+## Solutions
+### Agile
+There are several approaches that can be taken in the context of Agile depending on who's listening. The following main concepts are from [Wikipedia](https://en.wikipedia.org/wiki/Agile_software_development)
+* *Incremental and evolutionary processes* - Submitting pull requests for early feedback, or creating an issue for early stakeholder discussion aids this process, allowing ideas to be shaped and defined in the very early conceptual stages, thus being able to nourish good ideas and filter out unfeasible ideas before there are any significant costs associated with them.
+* *Efficient and face-to-face communication* - This part will be difficult as communications are not face to face. However they are efficient and tend to be very inclusive. It's possible to cite that this is how much of the software that runs the internet was successfully coordinated and built so there's some virtue to doing it the IS way, which is efficient but online communication. Another point of argument is that there's no better way to communicate your expectations than to actually implement it, making the process extremely effective
+* *Quality Focus* this is a major point for IS. Because of the expected submission by a variety of teams and individuals, extra care is taken to make sure that the submitted code is screened to be production quality. Comprehensive documentation for submitters, test coverage, CI/CD, static code analysis and other techniques are often use to screen code from strangers in the open source world, these techniques have great benefits for internal teams.
+
+##### [SCRUM](https://en.wikipedia.org/wiki/Scrum_%28software_development%29)
+Tshorten the example and make it more concrete for SCRUM, a popular agile methodology, one could cite the following scenario:
```

Comment:
_Tshorten_ => _To shorten_? I like the example listed here. It happens so often in big software companies which have more of an ownership culture and less of an IS culture.